docs/development-guide.md

```diff
@@ -6,7 +6,21 @@ This doc explains how to build and run the Online Boutique source code locally u
 
 - [Docker for Desktop](https://www.docker.com/products/docker-desktop)
 - [kubectl](https://kubernetes.io/docs/tasks/tools/) (can be installed via `gcloud components install kubectl` for Option 1 - GKE)
-- [skaffold **2.0.2+**](https://skaffold.dev/docs/install/) (latest version recommended), a tool that builds and deploys Docker images in bulk. 
+- [skaffold **2.0.2+**](https://skaffold.dev/docs/install/) (latest version recommended), a tool that builds and deploys Docker images in bulk.
+  
+  **Install skaffold:**
+  ```sh
+  # Linux
+  curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-amd64
+  chmod +x skaffold
+  sudo mv skaffold /usr/local/bin/
+  
+  # macOS
+  brew install skaffold
+  
+  # Or download from: https://skaffold.dev/docs/install/
+  ```
```

Comment:
I would rather not have these instructions here, since we'd be duplicating a source of truth that already exists (which is linked in the doc: https://skaffold.dev/docs/install/)